programs/steward/src/state/steward_state.rs

```diff
@@ -1026,9 +1017,12 @@ impl StewardState {
                     }
                 }
                 RebalanceType::Increase(amount) => {
-                    self.validator_lamport_balances[index] = self.validator_lamport_balances[index]
-                        .checked_add(amount)
-                        .ok_or(StewardError::ArithmeticError)?;
+                    if self.validator_lamport_balances[index] != LAMPORT_BALANCE_DEFAULT {
```

Comment:
Do we want this same check for Decrease?

Reply:
yep, good catch